visidata/cliptext.py

```diff
@@ -160,12 +162,17 @@ def _clipstr(s, dispw, trunch='', oddspacech='', combch='', modch=''):
             newc = c
             chlen = dispwidth(c)
 
+        #if the next character will not fit
         if dispw and w+chlen > dispw:
-            if trunchlen and dispw > trunchlen:
-                lastchlen = _dispch(ret[-1])[1]
-                if w+trunchlen > dispw:
+            if trunchlen > dispw:
+                vd.warning(f'truncator {repr(trunch)} is too big to fit in dispw of {dispw}')
```

Comment:
This is a very hot/inner routine, so we shouldn't show a warning here, nor even a debug.  Let's remove it.